precompiles/regular/testutil/RegularCaller.go

```diff
@@ -0,0 +1,286 @@
+// Code generated - DO NOT EDIT.
```

Comment:
we should look into excluding it from code coverage

Right now it seems it is included as there are the codecov comments